src/coreclr/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -1487,7 +1490,7 @@ void CodeGen::genSSE42Intrinsic(GenTreeHWIntrinsic* node)
 // Arguments:
 //    node - The hardware intrinsic node
 //
-void CodeGen::genAvxOrAvx2Intrinsic(GenTreeHWIntrinsic* node)
+void CodeGen::genAvxOrAvx2OrAvx512Intrinsic(GenTreeHWIntrinsic* node)
```

Comment:
Maybe `genAvxOrLaterIntrinsic` or `genAvxFamilyIntrinsic` ?

Reply:
Changed to `genAvxFamilyIntrinsic` - done